util/eventbus/subscribe.go

```diff
@@ -8,8 +8,19 @@ import (
 	"fmt"
 	"reflect"
 	"sync"
+	"time"
 )
 
+type queuedEvent struct {
+	Event              any
+	From               *Client
```

Comment:
Is this "from" or "via"? (I guess it doesn't matter, now that the client handles both ends of the transaction)